react/Columns/Columns.less

```diff
@@ -20,11 +23,17 @@
     .columns_tight & {
       padding: 0 (@gutter-width / 4);
     }
-    &:first-child {
+    .columns:not(.columns_reverse) &:first-child {
```

Comment:
Could the duplicate statements be combined? Using `,` to combine them.

Reply:
Done